dbt/models/reporting/schema.yml

```diff
@@ -235,6 +235,15 @@ models:
               (year >= '2021' OR (
               year = '2020' AND stage_name = 'BOR CERTIFIED')
               )
+      - is_null:
+          name: reporting_vw_pin_value_long_exempt_parcels_values
+          column_name: pin
+          config:
+            tags:
+              - test_qc_iasworld
```

Comment:
Not sure iasworld is the correct tag, but we're really only worried about the underlying data here.